rtcstats.js

```diff
@@ -171,31 +171,39 @@ module.exports = function(trace, getStatsInterval, prefixesToWrap) {
       // TODO: do we want one big interval and all peerconnections
       //    queried in that or one setInterval per PC?
       //    we have to collect results anyway so...
+      var getStats = function() {
+        if (isFirefox || isSafari) {
+          pc.getStats(null).then(function(res) {
+            var now = map2obj(res);
+            var base = JSON.parse(JSON.stringify(now)); // our new prev
+            trace('getstats', id, deltaCompression(prev, now));
```

Comment:
I thought it was `getStats` camelCase

Reply:
see the server, getStats and getstats have very different semantics